modules/standard/FileSystem.chpl

```diff
@@ -172,10 +172,10 @@ proc locale.chdir(out error: syserr, name: string) {
    :arg name: The intended current working directory
    :type name: `string`
 */
-proc locale.chdir(name: string) {
+proc locale.chdir(name: string) throws {
   var err: syserr = ENOERR;
   chdir(err, name);
-  if err != ENOERR then ioerror(err, "in chdir", name);
+  if err != ENOERR then try ioerror(err, "in chdir", name);
```

Comment:
I'm a little surprised this is a `try` instead of a direct `throw`, though I'm assuming ioerror is a function that throws?

Reply:
I say that because the intention of those statements is that if we enter the `then` block, we are in an error situation and so it would be problematic not to `throw` if that's the behavior we should perform

Reply:
Right, `ioerror()` is a function that `throws` if an `err` is passed to it. It was easier to change the code in `ioerror` once rather than replace all calls with a `throw`, at least for now.